score.rb

```diff
@@ -4,7 +4,7 @@ class Score
   WINNING_SCORE = 11
   MIN_DIFFERENCE = 2
 
-  WINNING_SET_SCORE = 2
+  WINNING_SET_SCORE = 3
```

Comment:
Ah okay, you’re right. But then the commit message should have mentioned why we’re making this (seemingly unrelated) change :)